sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -637,8 +636,6 @@ export type TransferProgressEvent = {
 
 // @public
 export interface TryAddOptions {
-    // @deprecated (undocumented)
-    parentSpan?: Span | SpanContext | null;
```

Comment:
minor version bump probably enough for this

Reply:
agreed